solidity/contracts/libs/Call.sol

```diff
@@ -4,51 +4,109 @@ pragma solidity ^0.8.13;
 import {Address} from "@openzeppelin/contracts/utils/Address.sol";
 
 library CallLib {
+    struct StaticCall {
+        address to;
+        bytes data;
+    }
+
     struct Call {
         address to;
+        uint256 value;
         bytes data;
     }
 
-    function _multicall(address to, bytes[] memory calls) internal {
+    struct StaticCallWithCallback {
+        StaticCall _call;
+        bytes callback;
+    }
+
+    function call(Call memory _call)
+        internal
+        returns (bytes memory returnData)
+    {
+        return Address.functionCallWithValue(_call.to, _call.data, _call.value);
+    }
+
+    function staticcall(StaticCall memory _call)
```

Comment:
Btw where is this used? Do we ever do static calls without callbacks?

I guess maybe we will later if we implement resolve/reject?

Reply:
making private for now